azavea/climate-change-lab
# Feature/awf/chart components to lib#283

## Overview

Removes all of the components added in azavea/climate-change-components#7 and replaces them with their equivalents from the library.

### Demo

N/A should look and function the same as before.


### Notes

A few miscellaneous commits that cleanup unused modules/code/dependencies.

Will not merge until companion PR is merged and the components lib version can be updated.

## Testing Instructions

See azavea/climate-change-components#7 for testing instructions, they're the same for this PR

## Checklist

Closes #283
